WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1338,7 +1454,7 @@ static GSourceFuncs _handlerIntervention =
 
             // Disk Cache
             if (_config.DiskCache.Value().empty() == false)
-                Core::SystemInfo::SetEnvironment(_T("WPE_DISK_CACHE_SIZE"), _config.DiskCache.Value(), !environmentOverride);
```

Comment:
This is not the same as what diskCacheDir was intended to do. please keep the original as it was.